src/material/button/button-base.ts

```diff
@@ -95,6 +103,7 @@ export class MatButtonBase
     elementRef: ElementRef,
     public _platform: Platform,
     public _ngZone: NgZone,
+    private _focusMonitor: FocusMonitor,
```

Comment:
You should use the new `inject` function to get the focus monitor. That way you can avoid all the constructor changes.

Reply:
Great point - done.